app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
     </p>
   {% endif %}
 
-  {{ summary.heading("Draft services") }}
+  <h2 class="govuk-heading-m">Draft services</h2>
   {% if framework.status == 'open' %}
-    {{ summary.top_link("Add a service", url_for(".start_new_draft_service", framework_slug=framework.slug, lot_slug=lot.slug)) }}
-  {% elif framework.status == 'pending' %}
-    <p class="hint">These services were not completed</p>
-  {% endif %}
-  {% call(draft) summary.list_table(
-    drafts,
-    caption="Draft services",
-    empty_message="You haven’t added any services yet." if framework.status == 'open' else "You didn’t add any services.",
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(draft.serviceName,
-                              url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)) }}
+  {% set ns = namespace(draft_rows = []) %}
+  {% for draft in drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+    {{ govukButton({
+      "text": "Make a copy",
```

Comment:
```suggestion
      "html": 'Make a copy <span class="govuk-visually-hidden">of {{ draft.serviceName }}</span>',
```

Reply:
I had to do this a different way to make Jinja happy, but should work as intended now after 37a37a3